modules/executive/components/DateFilter.tsx

```diff
@@ -59,7 +59,7 @@ export default function DateFilter(props): JSX.Element {
           />
         </Flex>
 
-        <Text>Before</Text>
+        <Text>Executive posted before:</Text>
```

Comment:
don't have a strong opinion, but feel this could be shortened to just `Posted Before` before and `Posted after`